product_customer_code/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=====================
+Product customer code
+=====================
+
+This module extends of module product and allows to have different codes
+for the same product according to the customer.
+
+**NOTE**: This module is overlapped by `product_supplierinfo_for_customer
+<../product_supplierinfo_for_customer>`_
```

Comment:
I dont think the URL can work all the time. You need to specify absolute path for both.